From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Dutta, Ranjan" <ranjan.dutta@intel.com>
Date: Tue, 10 Mar 2020 22:04:27 +0800
Subject: [PATCH] Revert "driver core: Print device when resources present in
 really_probe()"

This reverts commit 73504f095df344ad50be94c760d7c6188ad68c15.

Tracked-On: PKT-3178
---
 drivers/base/dd.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/base/dd.c b/drivers/base/dd.c
index caaeb7910..38a3107e0 100644
--- a/drivers/base/dd.c
+++ b/drivers/base/dd.c
@@ -470,10 +470,7 @@ static int really_probe(struct device *dev, struct device_driver *drv)
 	atomic_inc(&probe_count);
 	pr_debug("bus: '%s': %s: probing driver %s with device %s\n",
 		 drv->bus->name, __func__, drv->name, dev_name(dev));
-	if (!list_empty(&dev->devres_head)) {
-		dev_crit(dev, "Resources present before probing\n");
-		return -EBUSY;
-	}
+	WARN_ON(!list_empty(&dev->devres_head));
 
 re_probe:
 	dev->driver = drv;
-- 
https://clearlinux.org

